src/Avalonia.Controls/ListBoxItem.cs

```diff
@@ -48,14 +48,16 @@ protected override AutomationPeer OnCreateAutomationPeer()
 
         protected override void OnPointerPressed(PointerPressedEventArgs e)
         {
+            var handled = e.Handled; // need to ignore context menu detection
+
             base.OnPointerPressed(e);
 
             _pointerDownPoint = s_invalidPoint;
 
-            if (e.Handled)
+            if (handled)
```

Comment:
Why should we ignore `Handled` set by `base.OnPointerPressed`?

Reply:
Because new OnPointerPressed in base Control sets e.Handled = true in case Ctrl+Click was pressed, but for lists opening context menu should not prevent selecting the item.

Reply:
Ctrl+Click in ListBox means "toggle selection for an item without changing the rest of the selection"

Reply:
So with proposed code here it would trigger both the context menu event _and_ selection

Reply:
On macos toggling is performed with Command+Click, so there is no conflict.
Currently opening context menu with right button also selects item, so I thought that it is desired behavior and kept it.